scaffold.md

```diff
@@ -6,8 +6,42 @@ In this wave we will add the first set of user interactivity and persistence.
     - (done) After the new task is added, the site should take the user back to the home page which displays the full list of tasks. 
     - (done) The new task that was just added should be included in the full list of tasks.
 
-  - Be able to delete an existing task:
-    - Add a route and controller action whose responsibility is deleting a task (RESTful routes)
-    - On the home page, add a button or link for each task that will, once clicked...
```

Comment:
This is great! I like that you made sure that you had a plan and used this as a checklist